src/hnsw/options.c

```diff
@@ -178,7 +178,7 @@ void _PG_init(void)
     add_int_reloption(ldb_hnsw_index_withopts,
                       "m",
                       "HNSW M hyperparameter",
-                      HNSW_DEFAULT_M,
```

Comment:
My understanding is you changed the default values to 0 (which is an invalid value of all of these), to later require explicit passing of these parameters when creating an index from a file path. Is this the case?
Does this affect the printed index parameters when a regular `hnsw` index is created with default parameters and when you run `\di` or `\d`on the table to see all the indexes and their creation parameters?

Reply:
Actually I have changed them to be able to determine if they were passed or not via comparing to NULL values, but think I will close this PR and throw and error if the file won't exist asking to use `lantern_reindex_external_index()` function from this [PR](https://github.com/lanterndata/lantern/pull/250).